Project_0.Altis/Templates/Loadouts/RHS_AAF2017/AAF_Elite/AAF_elite_AA.sqf

```diff
@@ -9,23 +9,21 @@ removeGoggles this;
 
 _RandomVest = ["FGN_AAF_CIRAS_RF01","FGN_AAF_CIRAS_RF01_Belt","FGN_AAF_CIRAS_RF01_Belt_CamB","FGN_AAF_CIRAS_RF01_CamB"] call BIS_fnc_selectRandom;
 this addVest _RandomVest;
-_RandomHeadgear = ["FGN_AAF_PASGT_Type07","FGN_AAF_PASGT_Type07_ESS","FGN_AAF_PASGT_Type07_ESS_2"] call BIS_fnc_selectRandom;
+_RandomHeadgear = ["FGN_AAF_PASGT","FGN_AAF_PASGT_ESS","FGN_AAF_PASGT_ESS_2","FGN_AAF_PASGT_Type07","FGN_AAF_PASGT_Type07_ESS","FGN_AAF_PASGT_Type07_ESS_2"] call BIS_fnc_selectRandom;
```

Comment:
Use selectRandom instead of BIS_fnc_selectRandom, it's faster.
https://community.bistudio.com/wiki/selectRandom